handlers/boilerplatecode.ts

```diff
@@ -0,0 +1,11 @@
+import { Handler, Context } from "aws-lambda";
+import { download } from "../services/compileGRPCStubs";
+
+export const create: Handler = async (
```

Comment:
wrap the return type with Promise e.g. `Promise<Handler>` for async methods

Reply:
Added in the latest commit